runtime/lang/menu_ja_jp.utf-8.vim

```diff
@@ -1,315 +1,1266 @@
-menutrans Select\ &All		すべて選択(&A)
-
-" The GUI toolbar (for Win32 or GTK)
+menutrans &Window ウィンドウ(&W)
+" Window menuitems {{{1
+menutrans &New<Tab>^Wn 新規作成(&N)<Tab>^Wn
+menutrans S&plit<Tab>^Ws 分割(&P)<Tab>^Ws
+menutrans Sp&lit\ To\ #<Tab>^W^^ 裏バッファへ分割(&L)<Tab>^W^^
+menutrans Split\ &Vertically<Tab>^Wv 垂直分割(&V)<Tab>^Wv
+menutrans Split\ File\ E&xplorer ファイルエクスプローラ(&X)
+menutrans &Close<Tab>^Wc 閉じる(&C)<Tab>^Wc
+menutrans Close\ &Other(s)<Tab>^Wo 他を閉じる(&O)<Tab>^Wo
+menutrans Move\ &To 移動(&T)
+menutrans &Top<Tab>^WK 上(&T)<Tab>^WK
+menutrans &Bottom<Tab>^WJ 下(&B)<Tab>^WJ
+menutrans &Left\ Side<Tab>^WH 左(&L)<Tab>^WH
+menutrans &Right\ Side<Tab>^WL 右(&R)<Tab>^WL
+menutrans Rotate\ &Up<Tab>^WR 上にローテーション(&U)<Tab>^WR
+menutrans Rotate\ &Down<Tab>^Wr 下にローテーション(&D)<Tab>^Wr
+menutrans &Equal\ Size<Tab>^W= 同じ高さに(&E)<Tab>^W=
+menutrans &Max\ Height<Tab>^W_ 最大高に(&M)<Tab>^W_
+menutrans M&in\ Height<Tab>^W1_ 最小高に(&i)<Tab>^W1_
+menutrans Max\ &Width<Tab>^W\| 最大幅に(&W)<Tab>^W\|
+menutrans Min\ Widt&h<Tab>^W1\| 最小幅に(&H)<Tab>^W1\|
+" }}}
+
+" The popup menu {{{1
+menutrans &Undo 取り消す(&U)
+menutrans Cu&t 切り取り(&T)
+menutrans &Copy コピー(&C)
+menutrans &Paste 貼り付け(&P)
+menutrans &Delete 削除(&D)
+menutrans Select\ Blockwise 矩形ブロック選択
+menutrans Select\ &Word 単語選択(&W)
+menutrans Select\ &Sentence 文選択(&S)
+menutrans Select\ Pa&ragraph 段落選択(&R)
+menutrans Select\ &Line 行選択(&L)
+menutrans Select\ &Block ブロック選択(&B)
+menutrans Select\ &All すべて選択(&A)
+
+" func! <SID>SpellPopup()
+if !exists("g:menutrans_spell_change_ARG_to")
+  let g:menutrans_spell_change_ARG_to = '"%s"\ を変更'
+endif
+if !exists("g:menutrans_spell_add_ARG_to_word_list")
+  let g:menutrans_spell_add_ARG_to_word_list = '"%s"\ を単語リストに追加'
+endif
+if !exists("g:menutrans_spell_ignore_ARG")
+  let g:menutrans_spell_ignore_ARG = '"%s"\ を無視'
+endif
+" }}}
+
+" The GUI toolbar {{{1
 if has("toolbar")
   if exists("*Do_toolbar_tmenu")
     delfun Do_toolbar_tmenu
   endif
   fun Do_toolbar_tmenu()
-    tmenu ToolBar.Open		ファイルを開く
-    tmenu ToolBar.Save		現在のファイルを保存
-    tmenu ToolBar.SaveAll	すべてのファイルを保存
-    tmenu ToolBar.Print		印刷
-    tmenu ToolBar.Undo		取り消し
-    tmenu ToolBar.Redo		もう一度やる
-    tmenu ToolBar.Cut		クリップボードへ切り取り
-    tmenu ToolBar.Copy		クリップボードへコピー
-    tmenu ToolBar.Paste		クリップボードから貼り付け
-    tmenu ToolBar.Replace	検索 / 置換...
-    tmenu ToolBar.FindNext	次を検索
-    tmenu ToolBar.FindPrev	前を検索
-    if 0	" disabled; These are in the Windows menu
-      tmenu ToolBar.New		新規ウィンドウ作成
-      tmenu ToolBar.WinSplit	ウィンドウ分割
-      tmenu ToolBar.WinMax	ウィンドウ最大化
-      tmenu ToolBar.WinMin	ウィンドウ最小化
-      tmenu ToolBar.WinClose	ウィンドウを閉じる
+    let did_toolbar_tmenu = 1
+    tmenu ToolBar.Open ファイルを開く
+    tmenu ToolBar.Save 現在のファイルを保存
+    tmenu ToolBar.SaveAll すべてのファイルを保存
+    tmenu ToolBar.Print 印刷
+    tmenu ToolBar.Undo 取り消し
+    tmenu ToolBar.Redo もう一度やる
+    tmenu ToolBar.Cut クリップボードへ切り取り
+    tmenu ToolBar.Copy クリップボードへコピー
+    tmenu ToolBar.Paste クリップボードから貼り付け
+    if !has("gui_athena")
```

Comment:
gui_athena has been removed. So, this might not be needed.

Reply:
> gui_athena has been removed. So, this might not be needed.

Thanks, ok, I will remove it.